tests/acceptance/features/bootstrap/OccContext.php

```diff
@@ -1225,6 +1225,36 @@ public function theFollowingLocalStoragesShouldNotExist(TableNode $mountPoints)
 		}
 	}
 
+	/**
+	 * @Then the following local storage should be listed:
+	 *
+	 * @param TableNode $table
+	 *
+	 * @return void
+	 * @throws Exception
+	 */
+	public function theFollowingLocalStorageShouldBeListed(TableNode $table) {
+		$expectedLocalStorages = $table->getColumnsHash();
+		$commandOutput = \json_decode($this->featureContext->getStdOutOfOccCommand());
+		foreach ($expectedLocalStorages as $expectedStorageEntry) {
+			$isStorageEntryListed = false;
+			foreach ($commandOutput as $listedStorageEntry) {
+				if ($expectedStorageEntry["MountPoint"] === $listedStorageEntry->mount_point) {
+					Assert::assertEquals($expectedStorageEntry['Storage'], $listedStorageEntry->storage);
```

Comment:
```suggestion
					Assert::assertEquals($expectedStorageEntry['Storage'], $listedStorageEntry->storage, "Storage column does not have the expected value");
```
It is nice to get a message about which column has the problem, rather than just `Failed asserting that two strings are equal.`

Add messages to each `Assert` call.

Reply:
fixed